App/screens/CredentialOffer.tsx

```diff
@@ -29,21 +35,69 @@ const styles = StyleSheet.create({
   },
 })
 
+const INDY_CREDENTIAL_KEY = '_internal/indyCredential'
```

Comment:
Should this go in constants? If its just used in once place probably not - just checking.

Reply:
From what I saw in AFJ, this is likely just temporary.